airflow/www/views.py

```diff
@@ -251,16 +252,21 @@ def get_int_arg(value, default=0):
             elif cookie_val:
                 arg_tags_filter = cookie_val.split(',')
 
+        if request.args.get('status') is None:
```

Comment:
```suggestion
        if arg_status_filter is None:
```

We've already got it once, no need to ask for it again.

Reply:
Updated.